packages/docusaurus-theme-search-algolia/src/theme/SearchBar/styles.css

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+:root {
+  --docsearch-primary-color: var(--ifm-color-primary);
```

Comment:
@lex111 This will always sync the Docusaurus primary color with DocSearch. Do you think that this is appropriate?

Reply:
Yes, why not?

Reply:
Nice, just checking!